pkg/config/config_test.go

```diff
@@ -32,3 +32,25 @@ func TestUnknownConfigFields(t *testing.T) {
 		require.Contains(t, err.Error(), "field UnknownConfigurationField not found in type config.Config")
 	})
 }
+
+func TestLoadFileWithMissingDefaultConfigPath(t *testing.T) {
+	originalPath := DefaultConfigPath
+	temporaryPath := filepath.Join(filepath.Dir(originalPath), "missing-config")
+
+	if _, err := os.Stat(originalPath); os.IsNotExist(err) {
+		err = os.MkdirAll(originalPath, os.ModePerm)
+		require.NoError(t, err)
```

Comment:
You don't need this, it's always there. And if it's not due to the project refactoring, then just fail the test with the corresponding message.

Reply:
> We need a test for it. I'm not sure if it's possible, need to check: try to remove/rename the config directory in the start of the test (this change should be reverted in Cleanup) and then run the LoadFile with default config path and privnet config.

so this test is equal to call LoadFile('./config.protocol.privnet.yml') in an empty directory (at least without './config')?

The `TestLoadFileWithMissingDefaultConfigPath` is in `pkg/config/config_test.go`  so from this point the `config` directory is not `./config` but "./../../config". We do not need to clean up or create a directory in this case.

Reply:
>  so from this point the config directory is not ./config but "./../../config"

OK, then we can't trigger embedded config usage from this test. Then this test is useless, thus let's just remove it.

Reply:
OK, I see you've adjusted it so now it just loads config from file. Let it be.